.gitignore

```diff
@@ -28,6 +28,7 @@
 /doc/aspect.tag
```

Comment:
Please put these changes in a separate pull request. They are small but not related to the rest.